config/software/openssl.rb

```diff
@@ -17,135 +16,137 @@
-                        "-Wl,-blibpath:#{install_dir}/embedded/lib:/usr/lib:/lib"].join(" ")
-                      when "mac_os_x"
-                        ["./Configure",
-                         "darwin64-x86_64-cc",
-                         common_args,
-                        ].join(" ")
-                      when "smartos"
-                        ["/bin/bash ./Configure",
-                         "solaris64-x86_64-gcc",
-                         common_args,
-                         "-L#{install_dir}/embedded/lib",
-                         "-I#{install_dir}/embedded/include",
-                         "-R#{install_dir}/embedded/lib",
-                        "-static-libgcc"].join(" ")
-                      when "solaris2"
-                        if Config.solaris_compiler == "gcc"
-                          if architecture == "sparc"
-                            ["/bin/sh ./Configure",
-                             "solaris-sparcv9-gcc",
-                             common_args,
-                            "-L#{install_dir}/embedded/lib",
-                            "-I#{install_dir}/embedded/include",
-                            "-R#{install_dir}/embedded/lib",
-                            "-static-libgcc"].join(" ")
-                          else
-                            # This should not require a /bin/sh, but without it we get
-                            # Errno::ENOEXEC: Exec format error
-                            ["/bin/sh ./Configure",
-                             "solaris-x86-gcc",
-                             common_args,
-                            "-L#{install_dir}/embedded/lib",
-                            "-I#{install_dir}/embedded/include",
-                            "-R#{install_dir}/embedded/lib",
-                            "-static-libgcc"].join(" ")
-                          end
-                        else
-                          raise "sorry, we don't support building openssl on non-gcc solaris builds right now."
-                        end
-                      else
-                        ["./config",
-                        common_args,
-                        "disable-gost", # fixes build on linux, but breaks solaris
-                        "-L#{install_dir}/embedded/lib",
-                        "-I#{install_dir}/embedded/include",
-                        "-Wl,-rpath,#{install_dir}/embedded/lib"].join(" ")
-                      end
-
-  # openssl build process uses a `makedepend` tool that we build inside the bundle.
-  env["PATH"] = "#{install_dir}/embedded/bin" + File::PATH_SEPARATOR + ENV["PATH"]
-
-  # @todo: move into omnibus-ruby
-  has_gmake = system("gmake --version")
-
-  if has_gmake
-    env["MAKE"] = "gmake"
-    make_binary = "gmake"
+  ]
+
+  if windows?
+    configure_args << "zlib-dynamic"
   else
-    make_binary = "make"
+    configure_args << "zlib"
   end
-  ship_license "https://raw.githubusercontent.com/openssl/openssl/master/LICENSE"
```

Comment:
same comment here about using `ship_license` instead of `license`/`license_file`